app/protocols/hyper-protocol.js

```diff
@@ -3,8 +3,8 @@ const SDK = require('dat-sdk')
 
 module.exports = async function createHandler () {
   const sdk = await SDK()
-  const { Hyperdrive, resovleName } = sdk
-  const fetch = datFetch({ Hyperdrive, resovleName, writable: true })
+  const { hyperdrive, resolveName } = sdk
+  const fetch = datFetch({ hyperdrive, resolveName, writable: true })
```

Comment:
Good catch with resovleName, but I think Hyperdrive needs the capital H in both cases. 😁

```suggestion
  const { Hyperdrive, resolveName } = sdk
  const fetch = datFetch({ Hyperdrive, resolveName, writable: true })
```

https://github.com/RangerMauve/dat-fetch/blob/master/index.js#L20
https://github.com/datproject/sdk/blob/master/index.js#L91